### DISCUSSION TITLE
在网站中可以使用  @tauri-apps/api  吗

### DISCUSSION THREAD

**Discussion by iFTY-R** [tw93/Pake#570] [Q&A]
在网站中可以使用  @tauri-apps/api  吗
https://tauri.app/zh-cn/v1/api/js/

**Comment by tw93** [comment]
可以的 做一个判断 在 pake里面就好 可以通过 pakeToast 这个全局方法来判断

**Comment by iFTY-R** [comment]
window.__TAURI__ 通过这个也可以
